delft3dworker/models.py

```diff
@@ -311,26 +311,26 @@ class Scene(models.Model):
         # User input wait phase
         (6, 'idle', 'Idle: waiting for user input'),
 
-        # Workflow phases
+        # Main workflow phases
         (11, 'sim_start', 'Starting workflow'),
         (12, 'sim_run', 'Running workflow'),
         (13, 'sim_fin', 'Removing workflow'),
 
+        # Stopping phase
+        (20, 'stopping', 'Stopping workflow'),
+
         # Other phases
         (500, 'fin', 'Finished'),
         (501, 'fail', 'Failed'),
+        (503, 'stopped', 'Stopped')
     )
 
     phase = models.PositiveSmallIntegerField(default=phases.new, choices=phases)
 
-    # PROPERTY METHODS
-
-
-
     # UI CONTROL METHODS
 
     def reset(self):
-        if self.phase == self.phases.fin:
+        if self.phase == (self.phases.fin or self.phases.stopped):
```

Comment:
Could we make this into a tuple of finished states? Like all 500+ in a tuple Scene.FINISHED or similar. We should also include 501, as this is now irreparable.